src/modules/Thread/components/ParentMessageInfo/index.tsx

```diff
@@ -42,8 +42,8 @@ export default function ParentMessageInfo({
     isOnline,
     userMention,
   } = config;
-  const userId = stores?.userStore?.user?.userId;
```

Comment:
Seems every objects' type here are not nullable. So removed all `?` but plz tell me if we need to be safe here. Then I'll correct each types as nullable :)

Reply:
Removing all `?` is okay ~

Reply:
userId could be undefined
so how about
```
const userId = stores.userStore.user?.userId;
```